packages/Search/src/details/DTK_DetailsTreeConstruction_decl.hpp

```diff
@@ -18,12 +18,15 @@
 #include <DTK_DetailsNode.hpp>
 #include <DTK_KokkosHelpers.hpp> // clz
 
+#include <DTK_LinearBVH_fwd.hpp>
```

Comment:
Why is this include directive not in the block with all the other `DTK_` headers?

Reply:
Fixed.